paddle/fluid/operators/test_send_nccl_id.cc

```diff
@@ -68,9 +74,13 @@ void StartServer() {
   server_thread.join();
 }
 
-TEST(SendNcclId, GrpcServer) {
+TEST(SendNcclId, RPCServer) {
   g_req_handler.reset(new detail::RequestSendHandler(true));
+#ifdef PADDLE_WITH_GRPC
```

Comment:
Can use a macro somewhere in the header to define the RPCServer type to save some lines, like:

```c++
#ifdef PADDLE_WITH_GRPC
#define RPCSERVER_T detail::AsyncGRPCServer
#else
#define RPCSERVER_T detail::AsyncBRPCServer
#endif
```

Then use `RPCSERVER_T`

Reply:
Done.